crates/xline/tests/it/tls_test.rs

```diff
@@ -0,0 +1,145 @@
+use std::{fs, iter, path::PathBuf};
+
+use etcd_client::ConnectOptions;
+use test_macros::abort_on_panic;
+use tonic::transport::{Certificate, ClientTlsConfig, Identity};
+use utils::config::{
+    AuthConfig, ClusterConfig, CompactConfig, LogConfig, MetricsConfig, StorageConfig, TlsConfig,
+    TraceConfig, XlineServerConfig,
+};
+use xline_client::types::kv::PutRequest;
+use xline_test_utils::{enable_auth, set_user, Cluster};
+
+#[tokio::test(flavor = "multi_thread")]
```

Comment:
When do we need to enable `multi_thread` for test?

Reply:
Drop the `Cluster` need `multi_thread`